<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>812</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新細明體</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>CBETA 檔案比對程式</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>10</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;CBETA &lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;UTF8&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt; 檔案比對程式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>比對檔案一：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edFileName1">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btGetFile1">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>比對檔案二：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edFileName2">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btGetFile2">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>結果檔案　：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edFileName3">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btGetFile3">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>忽略文字　：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edSkipWord">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>忽略正規式：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edSkipRe">
          <property name="font">
           <font>
            <family>新細明體</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>忽略指定選項</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="cbSkipItem_a">
             <property name="text">
              <string>忽略小寫英文字母 (a-z)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSkipItem_A">
             <property name="text">
              <string>忽略大寫英文字母 (A-Z)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSkipItem_d">
             <property name="text">
              <string>忽略數字 (0-9)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSkipItem_h">
             <property name="text">
              <string>忽略經文行首 T01n0001_p0001a01║</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSkipItem_p">
             <property name="text">
              <string>忽略全型新式標點</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSkipItem_s">
             <property name="text">
              <string>忽略半型空格</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSkipItem_S">
             <property name="text">
              <string>忽略全部空格 (包含半型空格, 全型空格及 tab )</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSkipItem_t">
             <property name="text">
              <string>忽略 &lt;...&gt; 角括號所包含的範圍</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>執行命令：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edRunCommand">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="rightMargin">
         <number>30</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btRun">
          <property name="text">
           <string>執行</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btExit">
          <property name="text">
           <string>離開</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edFileName1</tabstop>
  <tabstop>btGetFile1</tabstop>
  <tabstop>edFileName2</tabstop>
  <tabstop>btGetFile2</tabstop>
  <tabstop>edFileName3</tabstop>
  <tabstop>btGetFile3</tabstop>
  <tabstop>edSkipWord</tabstop>
  <tabstop>edSkipRe</tabstop>
  <tabstop>cbSkipItem_a</tabstop>
  <tabstop>cbSkipItem_A</tabstop>
  <tabstop>cbSkipItem_d</tabstop>
  <tabstop>cbSkipItem_h</tabstop>
  <tabstop>cbSkipItem_p</tabstop>
  <tabstop>cbSkipItem_s</tabstop>
  <tabstop>cbSkipItem_S</tabstop>
  <tabstop>cbSkipItem_t</tabstop>
  <tabstop>btRun</tabstop>
  <tabstop>btExit</tabstop>
  <tabstop>edRunCommand</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
